src/main/kotlin/org/rust/lang/core/grammar/rust.bnf

```diff
@@ -1117,6 +1117,8 @@ external gteq   ::= collapse GTEQ GT EQ       { name = ">=" }
 external ltlteq ::= collapse LTLTEQ LT LT EQ  { name = "<<=" }
 external ltlt   ::= collapse LTLT LT LT       { name = "<<" }
 external lteq   ::= collapse LTEQ LT EQ       { name = "<=" }
+external oror   ::= collapse OROR OR OR       { name = "||" }
+external or     ::= onlySingle OR             { name = "|" }
```

Comment:
So as the order of rules has changed  maybe we don't need `or` any more?

And, if we still need it, I bet it can be defined simply as

```
bit_or_bin_expr     ::= restricted_expr '|' !'|' restricted_expr
```